docs-2.0/4.deployment-and-installation/6.deploy-text-based-index/3.deploy-listener.md

```diff
@@ -0,0 +1,92 @@
+# Deploy Raft Listener for Nebula Storage service
+
+## Before you start
+
+Before you start using the full-text index, please make sure that you know the [restrictions](../../4.deployment-and-installation/6.deploy-text-based-index/1.text-based-index-restrictions.md).
+
+Full-text index data is written to the Elasticsearch cluster asynchronously. To use the full-text index search, you must run one or more nebula-storaged process as the Raft Listener (short for Listener). The listener is a separate process and receives requests from Nebula Graph clients. It manages the traffic of these requests to the database server.
```

Comment:
If "listener" is a proper noun, keeping its capitalization consistent may make it more understandable.